esinstall/src/index.ts

```diff
@@ -93,6 +93,12 @@ function resolveExportsMap(packageManifest: {exports: {}}, exportsProp: string)
     exportMapEntry?.require ||
     exportMapEntry;
 
+  // For now we are bailing on when this is an object, but in the future we should add support
+  // as Parcel seems to do something with an object here.
+  if (typeof exportMapValue === 'object') {
```

Comment:
nit: we should make sure that this isn’t an array here, e.g. `if (!Array.isArray(exportMapValue) && typeof exportMapValue === 'object')`

Reply:
Or just add a `TODO` note here to come back to later, because array support is probably a whole dumb thing

Reply:
Or change the condition to be not a string, since we always expect this to return a string at the moment.

Reply:
Yeah that’d work too. And yeah it should always be a string (again, barring arrays, but unclear if that’s even valid at the moment)